.github/workflows/build_and_test.yml

```diff
@@ -10,11 +10,11 @@ on:
       fastcdr_versions:
         description: 'Fast CDR branches to be used'
         required: false
-        default: '["1.1.x", "master"]'
+        default: '["master"]'
       fastdds_branch:
         description: 'Fast DDS branch to be used'
         required: false
-        default: 'master'
+        default: '3.0.x-devel'
   pull_request:
   push:
     branches:
```

Comment:
I think that either we remove the 'branches:' argument, or we change main -> 3.0.x